### ISSUE TITLE
Example of a hello-world app?

### ISSUE THREAD

**Issue by deinspanjer** [a-type/graphql-arangodb#6]
Wondering if you have any examples of hooking up a simple apollo express server to graphql-arangodb?

I appreciate all the docs regarding the directives and such, but there are still some things I'm unclear on such as how exactly the context would be passed in.

**Comment by a-type** [comment]
Hi @deinspanjer , sorry I must have missed this issue!

I'll see if I can pull together an example. I've mostly developed this in the context of a side project that I abandoned last year, but it does work!

In the meantime, if it helps anyone, the tests run a full integration on a real schema: [type definitions](https://github.com/a-type/graphql-arangodb/blob/master/src/__tests__/fixtures/typeDefs.ts), [resolvers](https://github.com/a-type/graphql-arangodb/blob/master/src/__tests__/queryTranslation.test.ts#L10), and [an example query on a nested object](https://github.com/a-type/graphql-arangodb/blob/master/src/__tests__/queryTranslation.test.ts#L100). Of course tests are harder to follow than a real example, but the tools used are the same (`makeExecutableSchema`, etc). The test suite is just directly executing example queries against the compiled GraphQL excutable schema.

**Comment by AdsonCicilioti** [comment]
Hey @a-type !! Nice Utility!

This tool looks incredible, but I have practical debts regarding the use of it. Does it work under Express, Foxx, ...? An example will be very welcome. Please bless us more this gift.

**Comment by mysticaltech** [comment]
Yes, I have similar question, could I for instance use it inside of a Next.js app, server side?

**Comment by a-type** [comment]
Yikes, sorry all, I once again got distracted from this project. I haven't worked on Arango stuff in a while. I'll take a look at making an example now.

@mysticaltech I think NextJS routes are essentially "serverless" functions, so yeah, should work fine. You can set up Apollo Server within a NextJS route if I remember correctly. This library should be compatible with Yoga or Apollo.

**Comment by mysticaltech** [comment]
Wow, fantastic, so glad to hear this! This is a real game changer then. Thank you!

**Comment by a-type** [comment]
Ok, I added a basic example app to the repo. It's a one-file server that uses graphql-yoga.

I ran it locally with the latest ArangoDB Docker image. It requires some database setup - I used [Migo](https://github.com/deusdat/arangomigo), and I put the migrations in a directory in the example.

I hope the example is fairly straightforward. For a more robust app codebase that uses this library, check out my abandoned startup idea [Toast](https://github.com/a-type/toast/services/toast-core).
